packages/contentstack-auth/package.json

```diff
@@ -4,27 +4,11 @@
 	"version": "1.0.0",
 	"author": "Contentstack",
 	"bugs": "https://github.com/contentstack/cli/issues",
-	"scripts": {
-		"build": "npm run clean && npm run compile",
-		"clean": "rm -rf ./lib && rm -rf tsconfig.build.tsbuildinfo",
```

Comment:
add the script back

Reply:
Updated latest source from relase branch